src/Routes.tsx

```diff
@@ -301,6 +316,11 @@ export function Routes() {
         component={WalletListBnbContainer}
         exact={true}
       />
+      <PrivateRoute
+        path={STAKER_DOT_PATH}
+        component={StakerDotPathContainer}
+        exact={true}
```

Comment:
```suggestion
        exact
```
it's essentially the same

Reply:
yes, we can write that, but in the whole file there are `exact={true}`, so I think, I should't change it

Reply:
it's time to start improving this code!)